paint.js

```diff
@@ -622,11 +624,21 @@ PaintCanvasMorph.prototype.undo = function () {
     }
 };
 
+PaintCanvasMorph.prototype.disableSmoothing = function(ctx) {
```

Comment:
I think we should just use the standard setting and wait for all the browsers to catch up...